lib/scope.js

```diff
@@ -301,9 +301,11 @@ module.exports = {
 
   load: async function ( scope, file_name, attempt_num ) {
     /* Try to go to the interview url a few times */
-    let attempt_timeout = 10 * 1000;
-    // End failed attempts a little before full timeout so that we can get a useful error message
-    let max_attempts = Math.floor(scope.timeout/(attempt_timeout - 100));
```

Comment:
I think I see what you're talking about here! I think this would be more correct if it said `Math.floor(scope.timeout/(attempt_timeout + 100));` (changing the minus to a plus). We want each attempt to be given more time than the `attempt_timeout` value, because we want to account for the time needed to give a useful error message. In any case, your current code also works.